INFO   :: 00:09:03: root                       FlatOntologyManager.py <_set_ontology>93 :  Loading ontology: ontology/ontologies/Films-rules.json
INFO   :: 00:09:03: root                            FlatOntologyManager.py <_set_db>109 :  Loading database: ontology/ontologies/Films-dbase.db
INFO   :: 00:09:03: root                                      pydial.py <initialise>374 :  Seed = 117
INFO   :: 00:09:03: root                                      pydial.py <initialise>375 :  Root = /home/casanova/master/s3/application_innovation/Application Interaction vocale/projet_robotparlant/PyDial
DIAL   :: 00:09:03: root                                   pydial.py <chat_command>1126 :  *** Chatting with policies []: ***
INFO   :: 00:09:03: root                                    Agent.py <_hand_control>453 :  Launching Dialogue Manager for domain: Films
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:03: root                                      RegexSemI.py <__init__>83 :  No rTYPE regex has been set.
WARNING:: 00:09:04: root                                      HDCPolicy.py <__init__>82 :  Policy file is given: , but policy type is set to hdc. Ignoring the given policy file and using hdc policy.
WARNING:: 00:09:04: root                            Agent.py <_logical_requirements>405 :  Enforcing hello() at first turn in singledomain system
WARNING:: 00:09:04: root                                             Texthub.py <run>87 :  NOTE: texthub is not using any error simulation at present.
DIAL   :: 00:09:04: root                                       Agent.py <start_call>170 :  >> NEW DIALOGUE SESSION. Number: 1
INFO   :: 00:09:04: root                                    Agent.py <_hand_control>470 :  Domain Films is both already running and has control
DIAL   :: 00:09:04: root                                      Agent.py <_print_turn>579 :  ** Turn 0 **
ERROR  :: 00:09:04: root                              ContextLogger.py <_exceptHook>263 :  Uncaught exception: <type 'exceptions.AttributeError'>( 'DiaAct' object has no attribute 'encode' )
          Traceback (most recent call last):
            File "/home/casanova/master/s3/application_innovation/Application Interaction vocale/projet_robotparlant/PyDial/pydial.py", line 1143, in <module>
              run()
            File "/home/casanova/.local/lib/python2.7/site-packages/scriptine/command.py", line 196, in autocmds
              add_verbosity_option=add_verbosity_option)
            File "/home/casanova/.local/lib/python2.7/site-packages/scriptine/command.py", line 105, in parse_and_run_function
              return function(*args[:len(required_args)], **kw)
            File "/home/casanova/master/s3/application_innovation/Application Interaction vocale/projet_robotparlant/PyDial/pydial.py", line 1130, in chat_command
              hub.run()
            File "/home/casanova/master/s3/application_innovation/Application Interaction vocale/projet_robotparlant/PyDial/Texthub.py", line 88, in run
              sys_act = self.agent.start_call(session_id='texthub_dialog')
            File "/home/casanova/master/s3/application_innovation/Application Interaction vocale/projet_robotparlant/PyDial/Agent.py", line 197, in start_call
              self._print_sys_act(sys_act)
            File "/home/casanova/master/s3/application_innovation/Application Interaction vocale/projet_robotparlant/PyDial/Agent.py", line 594, in _print_sys_act
              if self.traceDialog>1: print '   Sys > {}'.format(sys_act.encode('utf-8'))
          AttributeError: 'DiaAct' object has no attribute 'encode'
          
